query.cc

```diff
@@ -73,6 +73,33 @@ std::ostream& operator<<(std::ostream& out, const read_command& r) {
         << "}";
 }
 
+std::ostream& operator<<(std::ostream& out, const forward_request::reduction_type& r) {
+    out << "reduction_type{";
+    switch (r) {
+        case forward_request::reduction_type::count:
+            out << "count";
+            break;
+        default:
```

Comment:
It's better to move the default case out of the switch, so that the compiler errors out if we forget to extend it.

Reply:
^

Reply:
Sorry for forgetting to remove this, it's fixed in v5.